test/test_plugins.py

```diff
@@ -243,9 +243,8 @@ def import_task_created_event(self, session, task):
 
         logs = [line for line in logs if not line.startswith('Sending event:')]
         self.assertEqual(logs, [
-            'Album: {0}/album'.format(self.import_dir),
-            '  {0}'.format(self.file_paths[0]),
-            '  {0}'.format(self.file_paths[1]),
```

Comment:
Did you mean to modify this test? As far as I can tell, nothing should have changed functionally without any plugins loaded.